src/main/java/dtb/lego/truck/rest/component/events/entity/repositories/AccelerometerEventRepository.java

```diff
@@ -0,0 +1,13 @@
+package dtb.lego.truck.rest.component.events.entity.repositories;
```

Comment:
On the boundary, control, entity framework, the repositories are keep on the control